sirbot/plugins/slack/plugin.py

```diff
@@ -141,20 +141,57 @@ def on_message(
             pattern=pattern, handler=(handler, configuration), **kwargs
         )
 
-    def on_action(self, action, handler, name="*", wait=True):
+    def on_action(self, callback_id, handler, name="*", wait=True):
```

Comment:
I would keep `action` here. The only reason is to not break compatibility

Reply:
Ah, true.  Wasn't thinking about keyword args.